spec/spec.md

```diff
@@ -109,28 +133,145 @@ resources that may exist.
+presentation and credential will be indexed. In addition, any verifiable credential must contain the identifier of the
+[issuer][issuer] that could also be de-anonymized and tracked by the publication of the credential. Therefore,
+[holders][holder] are _RECOMMENDED_ not to publish verifiable credentials that contain personally identifying data or
+data that can be used to correlate and track them or the [issuer][issuer].
+
+### Selective Disclosure
+
+_This section is non-normative_
+
+A [holder][holder] might want to withhold information that is part of a Verifiable Credential they want to publish.
+[Issuers][issuer] are _RECOMMENDED_ to support selective disclosure in issued credentials to grant [holders][holder] the
+ability to make fine-grained decisions about the information will be publicly available.
+
+## Security Considerations
+
+_This section is non-normative_
+
+### Acceptable Use
+
+_This section is non-normative_
+
+A [holder][holder] has the power to publish any verifiable credential they are holding. Before including a credential in
+a linked verifiable presetation the [holder][holder] is _RECOMMENDED_ to consider the constraints that are imposed by
+the [issuer][issuer].
+
+[Issuers][issuer] are _RECOMMENDED_ to utilize the mechanisms provided by the verifiable credentials data model
+[[spec:VC-DATA-MODEL]] to express constraints for issued credenditals. For example, the properties `expirationDate` and
+`termsOfUse` can be used to control or limit the public linking of credentials.
+
+### Limiting Validity of Verifiable Presentations
+
+_This section is non-normative_
+
+Linked Verifeable Presentations are accessible and verifiable by anyone and they might be accessible at the given URL
+indefinitely. A [holder][holder] might therefore desire to limit the validity of a presentation. This can be achieved by
+utilizing the `expires` property that is an optional element of `proof` property as specified in
+[[VC-DATA-INTEGRITY]](#ref:VC-DATA-INTEGRITY).
+
+### Verification of Linked Presentations
+
+_This section is non-normative_
+
+Unlike the interactive exchange of credentials between [holder][holder] and [verifier][verifier], linked verifiable
+presentations are statically linked and published by the [holder][holder]. Any entity that knows the [holder's][holder]
+DID is able to discover, access and verify the linked presentations without interaction between [holder][holder] and
+[verifier][verifier]. This flow requires additional caution from the [holder][holder] to ensure the integrity,
+authenticity and currentness of linked presentations. In addition to the security mechanisms of verifiable credentials
+and presentations [[VC-DATA-INTEGRITY]](#ref:VC-DATA-INTEGRITY) the `challenge` and `domain` search parameters in the
+linked presentation URL can help the [holder][holder] to increase the level of protection.
```

Comment:
>  to increase the level of protection

how?

Reply:
In commit c1461a2b3772532f21b321b636a9f04cb019c8fe I tried to explain the usefulness of the parameters. How do you like it?